i18n/hi.yaml

```diff
@@ -0,0 +1,53 @@
+FEED_URL_NOT_PARSE: यूआरएल पार्स करने में असमर्थ
+FETCH_ERROR: त्रुटि मिला
+404: फ़ेच त्रुटि HTTP404 मिला
+403: HTTP403 त्रुटि प्राप्त करना निषिद्ध है
+ALREADY_SUB: आपने पहले ही इस फ़ीड की सदस्यता ले ली है
+DB_ERROR: एसक्यूएल त्रुटि
+DID_NOT_SUB: आपने इस फ़ीड की सदस्यता नहीं ली है
+SUB_USAGE: 'उपयोग: /sub <RSS URL> स्वचालित रूप से RSS फ़ीड का पता लगाना समर्थित है'
+UNSUB_USAGE: 'उपयोग： /unsub  <RSS URL>'
+ADMIN_ONLY: केवल व्यवस्थापकों के लिए सक्षम करें
+UNSUBTHIS_USAGE: 'उपयोग: /unsubthis करें यह उस फ़ीड से संदेश का उत्तर दें जिसे आप सीधे अनसब्सक्राइब करना चाहते हैं '
+SAME_NAME: एक फ़ीड का शीर्षक समान है，कृपया उपयोग करें /unsub करें
+PROCESSING: प्रसंस्करण हो रहा है, कृपया थोड़ी देर प्रतीक्षा करें
+SUB_SUCCESS: सफलतापूर्वक सदस्यता लें
+UNSUB_SUCCESS: सफलतापूर्वक सदस्यता समाप्त करें
+NOT_SUB: किसी फ़ीड की सदस्यता नहीं ली गई है
+SUB_LIST: सदस्यता
+ERROR_MANY_TIME: कई बार लाने में असमर्थ, हो सकता है आप इसे अनसब्सक्राइब करना चाहें
+OPML_PARSE_ERROR: ओपीएमएल को पार्स करने में असमर्थ
+NETWORK_ERROR: नेटवर्क त्रुटि
+IMPORT_SUCCESS: सफलतापूर्वक आयात करें
+FILESYSTEM_ERROR: फ़ाइल सिस्टम त्रुटि
+UNSUB_ALL_SUCCESS: सभी को सफलतापूर्वक अनसब्सक्राइब करें
+EXPORT: 'उपयोग: /export सब्सक्रिप्शन को ओपीएमएल प्रारूप में निर्यात करें'
+SEND_FILE_IMPORT: सदस्यताएँ आयात करने के लिए एक opml फ़ाइल भेजें
+RSS_USAGE: 'उपयोग: /rss आउटपुट सदस्यता सूची में लिंक दिखाने के लिए "कच्चा" जोड़ें'
+VIEW_ALL_USAGE: /viewall सभी सदस्यताएँ और ग्राहकों की संख्या देखें, लिंक दिखाने के लिए कच्चा जोड़ें
```

Comment:
command paramters should not be translated

Reply:
Explain where?

Reply:
raw

Reply:
> raw

thats how Hindi language looks and understandable.

Reply:
raw is a paramter read by the Bot, not human.